corehq/apps/accounting/migrations/0015_datamigration_email_list.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from corehq.sql_db.operations import HqRunPython
+
+
+def copy_emails_to_email_list(apps, schema_editor):
+    BillingContactInfo = apps.get_model('accounting', 'BillingContactInfo')
+
+    for contact_info in BillingContactInfo.objects.all():
+        if contact_info.emails:
+            contact_info.email_list = contact_info.emails.split(',')
+        else:
+            contact_info.email_list = []
+        contact_info.save()
```

Comment:
assuming there aren't too many of these so it's probably fine, but this could really do a number on the db if there are a significant number of records